subsys/bluetooth/services/ots/ots_client.c

```diff
@@ -1263,6 +1287,49 @@ static int oacp_write(struct bt_conn *conn, struct bt_otc_internal_instance_t *i
 
 	return err;
 }
+
+static int oacp_checksum(struct bt_conn *conn, struct bt_otc_internal_instance_t *inst,
+			 uint32_t offset, uint32_t len)
+{
+	int err;
+
+	LOG_DBG("oacp_checksum");
```

Comment:
aren't you getting a checkpatch failure from this?

Reply:
No, would checkpatch suppress by git reuse-message?

Reply:
Removed

Reply:
the standard form is just `LOG_DBG("")` since `LOG_DBG` will include the current function name. 
Checkpatch had a check that failed if the function name (here "oacp_checksum") was found in the log string. Maybe they temporarily disabled it, I remember there were a bunch of false-positives.